WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -1191,6 +1191,30 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
     func gutenbergCapabilities() -> [Capabilities: Bool] {
         let isFreeWPCom = post.blog.isHostedAtWPcom && !post.blog.hasPaidPlan
         let isWPComSite = post.blog.isHostedAtWPcom || post.blog.isAtomic()
+
+        // Disable Jetpack-powered editor features in WordPress app based on Features Removal coordination
+        if JetpackFeaturesRemovalCoordinator.generalPhase() == .four {
```

Comment:
There are two phases where Jetpack features should be removed:
* `four`
* `newUsers`

I suggest adding a helper to identify whether features should be removed or not. Something along the lines of `shouldRemoveJetpackFeatures`. This helper should return true if the general phase is `four` or `newUsers`. What do you think?

Reply:
Ah, true, thanks @hassaanelgarem for the call-up about the two phases 🙇 .

> I suggest adding a helper to identify whether features should be removed or not. Something along the lines of `shouldRemoveJetpackFeatures`. This helper should return true if the general phase is `four` or `newUsers`. What do you think?

Good idea, I'll work on adding a function to determine when the Jetpack features should be removed. Similarly, I'll replicate this suggestion in Android 👍 .

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/19691/commits/fb13609c2e1a70c0c457d7ab25776df028a681de.